src/assignments/scrabble-scorer.rst

```diff
@@ -0,0 +1,157 @@
+Assignment #2: Scrabble Scorer
+==============================
+
+To see how far you skills have come, we are going to give you a mission to update
+our *super* important Scrabble Scorer program. What, did you think you were going
+to work on Mars rover code already?
+
+The current Scrabble Scorer program will take in a word and return the Scrabble value for
+that word. Below are the requirements for the new version.
+
+
+Requirements
+------------
+
+#. Create an ``oldScoreKey`` object that contains the previous scoring system.
+#. Create a ``transform`` function which will return a ``newScoreKey`` object.
+#. Create ``scoringAlgorithms`` array to hold three scoring objects, which include the scoring functions.
+#. Create a ``initialPrompt`` function that asks user what score algorithm to use.
+#. Create a ``runProgram`` function to contain the starting point for your running program.
+
+Example Output
+^^^^^^^^^^^^^^
+::
+
+   Welcome to the Scrabble score calculator. Use ctrl+c to quit.
```

Comment:
Instead of using ctrl-c to stop the program, how about adding 'quit' option to the scoring selection menu?  Maybe option 3 could be 'end' or 'quit'.

Reply:
I'm not opposed to this. Let's wait and see if some of our QA students also have the same feedback.

Reply:
OK.